docs/sources/panels/add-series-overrides.md

```diff
@@ -0,0 +1,43 @@
++++
+title = "Add series overrides"
+type = "docs"
+[menu.docs]
+parent = "panels"
+weight = 500
++++
+
+# Add series overrides
+
+> **Note:** This documentation refers to a feature only available in Grafana 7.0 beta.
```

Comment:
This is not true, what you are documenting here is available in Grafana for a while. It's Field options and Field options overrides that are available since 7.0